packages/eslint-plugin/src/rules/no-useless-template-literals.ts

```diff
@@ -7,12 +7,32 @@ import {
   getConstrainedTypeAtLocation,
   getParserServices,
   getStaticStringValue,
+  getWrappingFixer,
   isTypeFlagSet,
   isUndefinedIdentifier,
 } from '../util';
 
 type MessageId = 'noUselessTemplateLiteral';
 
+function getReplaceNodeText(
+  context: Readonly<TSESLint.RuleContext<'noUselessTemplateLiteral', []>>,
+  node: TSESTree.TemplateLiteral,
+  fixer: TSESLint.RuleFixer,
+): string | null {
+  const replaceResult = getWrappingFixer({
+    sourceCode: context.sourceCode,
+    node: node.expressions[0],
+    wrap: (...code: string[]) => {
+      return code.join('');
+    },
+  })(fixer);
```

Comment:
[Refactor] Coming from https://github.com/typescript-eslint/typescript-eslint/pull/8673#discussion_r1537842742:

> Instead of adding the overhead of the rest of `getWrappingFixer()`, suggestion: the specific text generation stuff should be extracted out and used here. Like, you could move the text generation from the `getWrappingFixer` into a separate function, and use that separate helper here.

I think this means that we should extract codegen part **from** `getWrappingFixer` implementation to the separate utility function, not extract whole `getWrappingFixer` call to the separate function. WDYT?

Reply:
I think I misunderstood the explanation. thank you The code generation part that generates parentheses was separated and made into a function.